src/include/daos_obj.h

```diff
@@ -586,9 +621,9 @@ daos_obj_fetch(daos_handle_t oh, daos_handle_t th, daos_key_t *dkey,
  *					aggregated. Update result is undefined.
  */
 int
-daos_obj_update(daos_handle_t oh, daos_handle_t th, daos_key_t *dkey,
-		unsigned int nr, daos_iod_t *iods, d_sg_list_t *sgls,
-		daos_event_t *ev);
+daos_obj_update(daos_handle_t oh, daos_handle_t th, uint32_t flags,
```

Comment:
64 bits will be placed on the stack for alignment here anyway so maybe uint64_t for forward compatibility?

Reply:
agreed, will update